wellcomeml/ml/__init__.py

```diff
@@ -33,3 +34,4 @@
 except ImportError as e:
     logger.error(e)
     logger.warning("Using WellcomeML without extras (transformers & torch).")
+    logger.debug(traceback.format_exc())
```

Comment:
That's great. We should in any case move towards a more granular approach with extras, but having the debug in the meantime is good!